### ISSUE TITLE
Import items

### ISSUE THREAD

**Issue by PetLahev** [PetLahev/VBE_Modules#2]
As a user I want to be able import an item to the active project

I have to be forced to choose a path from where the items should be loaded
but
If I already imported/exported to/from the project, the path should be defaulted for me.

A form with all items from the folder will be displayed adn I'll be able to
- select all
- select only certain types (class, module, form)
- select a specific item(s)

I need to be given with two options
- Overwrite if already exists (default)
- Make copy (the item with the same name in the project will be retained)

**Comment by rubberduck203** [comment]
I recently solved this problem. You may be interested in some of this code. 

https://github.com/retailcoder/Rubberduck/blob/1308c560626748c2922d56b995970feeda56a8e8/RetailCoder.VBE/Extensions/VBComponentsExtensions.cs#L24-L72

https://github.com/retailcoder/Rubberduck/blob/master/RetailCoder.VBE/Extensions/VbProjectExtensions.cs#L35-L89

The purpose of your project is also on our roadmap, so you're also welcome to contribute to our project. Either way, I'll be keeping an eye on this. =)

**Comment by PetLahev** [comment]
Excellent! Thanks Chris. I downloaded your project the other day and went through all the code (I was more than impressed, you both have done really great job) but I missed that one. 
Looks like I wrap your code with some UI and I'm done!
Thanks again

**Comment by rubberduck203** [comment]
You're welcome! Thank you for the bug reports!

**Comment by PetLahev** [comment]
The simple functionality works as expected so I'm closing it.
However I'd like to make it a bit better
- write code that will handle deleted paths
- validating project paths should be in other thread
- maybe some ++is component changed++ functionality?!

**Comment by rubberduck203** [comment]
You mentioned you'd like to watch for the module changing. If you figure that out please ping me. I'll do likewise. It's a problem that I've struggled to find a decent solution for.

**Comment by PetLahev** [comment]
Hi Chris
I'm not so ambitious (I'm still watching you guys! :) ) I saw your comments or discussion that you'd like to use some event from Windows or something like that. If I ever implement what I suggested, it will be very simple user driven functionality (probably a button like "Check changes") nothing complicated.
Anyway I'm still waiting for RubberDuck SC version and then I can throw away this one :)
